packages/server/src/routes/me/index.ts

```diff
@@ -64,17 +70,33 @@ export const sendInvite = (
       throw new MissingParamsError(["inviteEmail"]);
     }
 
+    if (videoToken && isJointVideo === undefined) {
+      throw new MissingParamsError(["isJointVideo"]);
+    }
+
+    // TODO generate this server side somewhere.
+    const inviteToken = videoToken;
+
+    if (inviteToken) {
```

Comment:
Would appreciate comment explaining that if inviteToken isn't present, it's coming from the web, and that that is soon to be deprecated/removed.